### PR TITLE
[CLI] Feature: Add "next" to run the next unsolved exercise.

### PR DESCRIPTION
**Pull request by jazzplato**
Based on #783 

This PR should allow users to use the following command to run the next unsolved exercise in the `exercises` vector.
```bash
rustlings run next
```

When there are no more unsolved exercises, it will print the following messages and `exit` with `1`.
```
🎉 Congratulations! You have done all the exercises!
🔚 There are no more exercises to do next!
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,6 +97,12 @@ In case you want to go by your own order, or want to only verify a single exerci
 rustlings run myExercise1
 ```
 
+Or simply use the following command to run the very next unsolved exerice in the tutorial:
```
Comment (fmoko) [inline README.md:100]:
```suggestion
Or simply use the following command to run the next unsolved exercise in the course:
```

**Comment by jazzplato** [comment]
I realized the `next` keyword can also help `hint` subcommand, so I updated the `README.md` to elaborate this feature in the doc.

**Review by fmoko — APPROVED** [review]
Thanks!

**Comment by fmoko** [comment]
@all-contributors please add @jazzplato for code

**Pull request merged**